content/raceday/heats.inc

```diff
@@ -1,3 +0,0 @@
-<h1>Heat Schedule</h1>
```

Comment:
This could maybe be cool to do something like the Leaderboard, but moved it to WP in planning to keep it a Google Sheet embed